src/Client.php

```diff
@@ -11,8 +11,12 @@ class Client
     const API_VERSION = '2.0';
     const APP_VERSION = 'pomelo-php-v2.1.2';
     const SIGN_METHOD = 'sha1';
-    const SANDBOX_ENDPOINT = 'https://api.dev.pomelopay.com/public/';
-    const PRODUCTION_ENDPOINT = 'https://api.pomelopay.com/public/';
+    const ENDPOINTS = [
+        'pomelopay' => ['sandbox'=> 'https://api.dev.pomelopay.com/public/', 'production' => 'https://api.pomelopay.com/public/'],
```

Comment:
I would instead of exposing all our clients / endpoints (business and security risk) allow you to manually provide the endpoint you want to use.